lib/notification_service/notification_service/server.py

```diff
@@ -115,10 +115,28 @@ def _init_server(self):
     def start(self, is_block=False):
         self._init_server()
         self.server.run(is_block)
+        if not is_block:
+            self._wait_for_server_available(timeout=self.config.wait_for_server_started_timeout)
 
     def stop(self):
         self.server.stop()
 
+    def _wait_for_server_available(self, timeout):
```

Comment:
Could the `timeout` be the millseconds?

Reply:
Yep, it can be float value to represent millsecsonds, I'll figure out in the comment.